machine/accton/accton_as4510_52t/busybox/patches/platform-as4510_52t.patch

```diff
@@ -0,0 +1,38 @@
+Patches for Accton AS4510_52T
+
+diff --git a/libbb/xfuncs_printf.c b/libbb/xfuncs_printf.c
+index d8a42ba..c52d78d 100644
+--- a/libbb/xfuncs_printf.c
++++ b/libbb/xfuncs_printf.c
+@@ -515,7 +515,13 @@ int FAST_FUNC bb_xioctl(int fd, unsigned request, void *argp, const char *ioctl_
+ 
+ 	ret = ioctl(fd, request, argp);
+ 	if (ret < 0)
+-		bb_simple_perror_msg_and_die(ioctl_name);
++	{
++#include <mtd/mtd-user.h>
++		if (request == MEMERASE)
++			bb_simple_perror_msg(ioctl_name);
```

Comment:
Why is it OK for MEMERASE to fail?  Seems strange to treat that request different from the rest.